scheduler.go

```diff
@@ -966,6 +966,22 @@ func (s *Scheduler) Tag(t ...string) *Scheduler {
 	return s
 }
 
+// GetAllTags returns all tags.
+func (s *Scheduler) GetAllTags() []string {
+	var tags []string
+	if s.tagsUnique {
+		s.tags.Range(func(key, value interface{}) bool {
+			tags = append(tags, key.(string))
+			return true
+		})
+	} else {
```

Comment:
with tags unique, the tags are still set on the jobs, so I think it would be cleaner to simply range over all the jobs and join the tags.

Reply:
13c4482c3aa0cc460fda878c9b0aee943dc29719